### DISCUSSION TITLE
FireStick 4K Unable to play 4K UHD?

### DISCUSSION THREAD

**Discussion by dizzwold** [awawa-dev/HyperHDR#978] [General]
Hi Guys,

 I've finally got my HyperSerialHDR up and running a very happy with it.

 I'm still finding my feet with it and got to find the time fully set things correctly.
 
 I went to watch a film last night on my ageing 4K FireStick and got the following message pop up;
Unable to play 4K UHD

 
![IMG_3753](https://github.com/user-attachments/assets/9ef6675a-8898-49f4-9d51-ec695fe75f4a)

 I have a new 2024 Samsung tv where (as far as I'm aware), all the HDMI ports are HDCP 2.2.
RPi 4B, Adafruit Scorpio, Kinovo 8K HDMI Splitter, and the following Grabber (Rybozen 4K);
https://www.amazon.co.uk/dp/B08JHNKDP8?ref=ppx_yo2ov_dt_b_fed_asin_title
 
 If I unplug the grabber and connect the HDMI cables together (taking the grabber out of the loop), go to play the same film, NO "Unable to play 4K" message.

 Stating the obvious is the grabber isn't what it claims to be "HDCP 2.2 EDID".

 I just wondered if anyone else has similar equipment or experienced this message?

**Comment by awawa-dev** [comment]
Hello
I think you are confusing "HDCP 2.2 EDID" (security standard) with the HDMI standard, e.g. HDMI2.0. Unfortunately, this is how experiments with unknown, cheap Chinese video equipment usually end - I also fell into similar traps a few times 😉

**Reply by dizzwold** [reply]
Hi @awawa-dev,

So is this a problem with the Grabber or the Splitter?

 If it's the grabber, what is the latest recommendation?

**Reply by awawa-dev** [reply]
You need HDMI splitter to workaround the HDCP. Are you sure that Kinovo 8K HDMI Splitter is a splitter or just a switch?

**Comment by awawa-dev** [comment]
BTW no modern grabber will allow you to capture HDCP protected stream. Your HDMI splitter must strip it.

**Reply by dizzwold** [reply]
Hi @awawa-dev,

 You are very much correct.

 Can you recommend a Splitter with 4 HDMI Inputs, and what is the latest recommended Grabber?

**Reply by awawa-dev** [reply]
Since you already have a switch then maybe just add a Ezcoo EZ-SP12H2? I cannot recommend any 4 ports spillter or matrix because I havent tested such device. Probably as always Ezcoo could have something interesting in this segment.

If you are not satisfied with the grabber, you can always replace it with a cheap ms2130 or take an interest in the latest game-changers that are being tested here: https://github.com/awawa-dev/HyperHDR/discussions/967#discussioncomment-11124448

**Comment by dizzwold** [comment]
@awawa-dev 

"Since you already have a switch then maybe just add a Ezcoo EZ-SP12H2?"

 So are you suggesting the output from the Kinovo switch goes to the EZ-SP12H2 and then to the grabber?

**Reply by awawa-dev** [reply]
yes, just to workaround HDCP
